app/bundles/PluginBundle/Command/PushLeadActivityCommand.php

```diff
@@ -0,0 +1,112 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2014 Mautic Contributors. All rights reserved.
+ * @author      Mautic
+ * @link        http://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+
+namespace Mautic\PluginBundle\Command;
+
+use Symfony\Bundle\FrameworkBundle\Command\ContainerAwareCommand;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Mautic\CoreBundle\Helper\DateTimeHelper;
+
+/**
+ * Class PushLeadActivityCommand
+ */
+class PushLeadActivityCommand extends ContainerAwareCommand
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('mautic:integration:pushleadactivity')
+            ->setAliases(
+                array(
+                    'mautic:integration:pushactivity',
+                    'mautic:pushactivity:integration'
+                )
+            )
+            ->setDescription('Push lead activity to integration.')
+            ->addOption(
+                '--integration',
+                '-i',
+                InputOption::VALUE_REQUIRED,
+                'Integration name. Integration must be enabled and authorised.',
+                null
+            )
+            ->addOption('--start-date', '-d', InputOption::VALUE_REQUIRED, 'Set start date for updated values.')
+            ->addOption(
+                '--end-date',
+                '-t',
+                InputOption::VALUE_REQUIRED,
+                'Set end date for updated values.'
+            )
+            ->addOption(
+                '--time-interval',
+                '-ti',
```

Comment:
Symfony won't like this as it'll throw a "Too many arguments" error. It has to be single letter that's not used.